README.md

```diff
@@ -1467,3 +1476,17 @@ The script serves the Swagger editor locally at a URL where your current version
 of the API documentation is visible. Change the docs in `azul/service/app.py`,
 save, and refresh the page and your changes will appear immediately along with
 any warnings or errors they may introduce.
+
+# 13. Onboarding
+ To assist with getting onboarded to the project, review the following documents:
+
+- [Azul Overview](#1-architecture-overview)  
+- [Getting Started](#2-getting-started)
+- [Contributing Guidelines](CONTRIBUTING.rst)
+  
+Request Access to Cloud Infrastructure and Tools:
+  - Slack [#team-boardwalk](https://ucsc-cgl.slack.com/archives/C705Y6G9Z) 
+  - Contact the Team Lead or Project Manager for access to the Azul GitHub Repository. 
```

Comment:
```suggestion
  - Push access to this repository
```

I think we should leave open who grants access 'cause that varies over time. We should generically instruct to ask on the Slack channel.